views/clucks/new.ejs

```diff
@@ -0,0 +1,11 @@
+<%- include("/Users/mikes/cluckr/views/partials/header.ejs") %> 
```

Comment:
@Mikecoolio If you set the path like this, I won't be able to import header.ejs because I don't use your environment. You need to use relative path to be able to import in everyone. This is the error `ENOENT: no such file or directory, open '/Users/mikes/cluckr/views/partials/header.ejs'`

Reply:
Oh no! I'm very sorry about the misunderstanding.  I never considered that.  I will use the relative path to import in everything in the future.  Thanks!